client/src/GlobalStyles.js

```diff
@@ -57,9 +59,17 @@ const GlobalStyles = () => {
         }
         h4 {
           font-family: "SFUIbold";
+          letter-spacing: -0.05em;
           font-size: 1em;
           color: #000000;
         }
+        menuLinks {
```

Comment:
You target a custom HTML element. I think this is a mistake :). 
Instead, you should create a styled component.

Like:
```jsx
const MenuLink = styled.a`
  font-family: "SFUIbold";
  letter-spacing: -0.05em;
  font-size: 2em;
  color: #000000;
  padding-bottom: 1em;
`;
```

```jsx
<MenuLink href="/offers">Angebote</MenuLink>
```

Reply:
I probably was way too deep into this 🤦 
Thank you!

Reply:
Fix #48